extra-node-modules.config.js

```diff
@@ -1,7 +1,5 @@
 /** @format */
 const enm = require( 'node-libs-react-native' );
-enm.fs = __dirname + '/__mocks__/nodejsMock.js';
-enm.net = __dirname + '/__mocks__/nodejsMock.js';
-enm.canvas = __dirname + '/__mocks__/nodejsMock.js';
+enm.canvas = null;
```

Comment:
I think this file is not used anywhere, anymore. Maybe we can delete it?

Reply:
That's true, thanks for pointing that out. I've removed `extra-node-modules.config.js` now.